sdcm/provision/network_configuration.py

```diff
@@ -89,6 +91,16 @@ def interface_ipv6_address(self):
         else:
             return None
 
+    @property
+    def device(self):
+        # Depend on network configuration:
+        # if broadcast_address.device_name if found
+        # else "eth0"
+        if address_config := [conf for conf in self.scylla_network_config if conf["address"] == "broadcast_address"]:
+            return "".join([ni.device_name for ni in self.network_interfaces if ni.device_index == address_config[0]['nic']]) \
+                   or "eth0"
+        return "eth0"
```

Comment:
Hardcoded default is problematic here, we should pick a default from a real existing interface.

Reply:
changed to "". If we did not recognized device name, do not put the value